LayoutTests/fast/forms/input-background-ua-media-query-expected.html

```diff
@@ -1,4 +1,4 @@
 <style>
 body { background-color: green }
 </style>
-<input value=foo style="background-color:white">
+<input value=foo style="border: 2px gray inset; background-color:white">
```

Comment:
Ideally each test change would be individually justified/explained in detail in the commit message.